napari/_qt/widgets/qt_welcome.py

```diff
@@ -172,3 +174,11 @@ def set_welcome_visible(self, visible=True):
     def resizeEvent(self, event):
         self.resized.emit()
         return super().resizeEvent(event)
+
+    def enterEvent(self, event):
+        self.enter.emit()
+        super().enterEvent(event)
+
+    def leaveEvent(self, event):
+        self.leave.emit()
+        super().leaveEvent(event)
```

Comment:
just reading the code it's not immediately obvious to me what these do - would a name like `cursorEntryEvent` be more descriptive? Otherwise just a docstring explaining would be good

Reply:
This functions are overload of Qt functions 
https://doc.qt.io/qt-5/qwidget.html#leaveEvent
https://doc.qt.io/qt-5/qwidget.html#enterEvent
so we cannot change names. Calling of super functions is not enough information that it comes from Qt?

Reply:
Thanks for explaining and sorry for the noise - it should have been enough info for me but I'm apparently a little slow this morning 😄 

Would you mind adding a small docstring anyway? Something like

```python
"""Emit our own events on entering/leaving the canvas."""
```